test/BasicClassesTest.php

```diff
@@ -86,6 +86,7 @@ public function testConstructor(): void
 
     public function testSetters(): void
     {
+        /** @var Success<mixed> $result */
```

Comment:
Required here, otherwise psalm sets the template to NULL, and rejects ->setData("bar").